linode/manager
# chore: [M3-6138] - Use Akamai Favicon

## Description 📝

- Uses Akamai Favicon

## Preview 📷

![Screenshot 2023-04-12 at 10 22 52 AM](https://user-images.githubusercontent.com/115251059/231487844-de451995-e771-4d90-a7f9-8313f5f1dd9a.jpg)

![Screenshot 2023-04-12 at 10 23 24 AM](https://user-images.githubusercontent.com/115251059/231487991-68729d6a-679f-4ca8-8933-6fc81064b956.jpg)


## How to test 🧪

- Spin up local dev server
- Visit `http://localhost:3000/` and verify you see the new Akamai favicon
